data/mods/Magiclysm/enchanted/enchanted_rings.json

```
@@ -0,0 +1,328 @@
+    "name_plural": "rings of perception +1",
+    "description": "A copper ring that makes you a little more perceptive when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 1 } ] } ] }
+  },
+  {
+    "copy-from": "mring_silver",
+    "type": "TOOL_ARMOR",
+    "id": "mring_perception_2",
+    "name": "ring of perception +2",
+    "name_plural": "rings of perception +2",
+    "description": "A silver ring that makes you a good bit more perceptive when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 2 } ] } ] }
+  },
+  {
+    "copy-from": "mring_gold",
+    "type": "TOOL_ARMOR",
+    "id": "mring_perception_3",
+    "name": "ring of perception +3",
+    "name_plural": "rings of perception +3",
+    "description": "A golden ring that makes you eye-openingly more perceptive when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 3 } ] } ] }
+  },
+  {
+    "copy-from": "mring_platinum",
+    "type": "TOOL_ARMOR",
+    "id": "mring_perception_4",
+    "name": "ring of perception +4",
+    "name_plural": "rings of perception +4",
+    "description": "A platinum ring that makes you much more perceptive when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 4 } ] } ] }
+  },
+  {
+    "copy-from": "mring_copper",
+    "type": "TOOL_ARMOR",
+    "id": "mring_speed_3",
+    "name": "ring of speed +3",
+    "name_plural": "rings of speed +3",
+    "description": "A copper ring that makes you a little faster when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 3 } ] } ] }
+  },
+  {
+    "copy-from": "mring_silver",
+    "type": "TOOL_ARMOR",
+    "id": "mring_speed_5",
+    "name": "ring of speed +5",
+    "name_plural": "rings of speed +5",
+    "description": "A silver ring that makes you a good bit faster when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 5 } ] } ] }
+  },
+  {
+    "copy-from": "mring_gold",
+    "type": "TOOL_ARMOR",
+    "id": "mring_speed_7",
+    "name": "ring of speed +7",
+    "name_plural": "rings of speed +7",
+    "description": "A golden ring that makes you quite hasty when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 7 } ] } ] }
+  },
+  {
+    "copy-from": "mring_platinum",
+    "type": "TOOL_ARMOR",
+    "id": "mring_speed_10",
+    "name": "ring of speed +10",
+    "name_plural": "rings of speed +10",
+    "description": "A platinum ring that makes you much quicker when you wear it.",
```

```suggestion
    "description": "A platinum ring that makes you much faster when you wear it.",
```
the other 4 rings have symmetry, but this one doesn't